server/shared/src/main/scala/org/http4s/server/websocket/WebSocketBuilder2.scala

```diff
@@ -79,6 +107,9 @@ sealed abstract class WebSocketBuilder2[F[_]: Applicative] private (
   def withFilterPingPongs(filterPingPongs: Boolean): WebSocketBuilder2[F] =
     copy(filterPingPongs = filterPingPongs)
 
+  def withDefragFrame(defragFrame: Boolean): WebSocketBuilder2[F] =
```

Comment:
More name bike-shedding 😅 🤔 I wonder if `withDefragment` or `withDefragmenting` will be nicer?

Reply:
I chose `withDefragment`, the simplest one to me.


[9f218f8](https://github.com/http4s/http4s/pull/7091/commits/9f218f886231816e84a198b2034a3bb1f1d7d7f1)